v1.0/v1.0/single_file_from_dir.cwl

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: Workflow
+
+requirements:
+  StepInputExpressionRequirement: {}
+
+inputs:
+  inf: File  # tarfile
+outputs:
+  output:
+    type: File
+    outputSource: second/output
+
+steps:
+  first:
+    run: dir3.cwl
+    in:
+      inf: inf
+    out: [ outdir ]
+  second:
+    run: cat-tool.cwl
+    in:
+      file1:
+        source: first/outdir
+        valueFrom: $(self.listing[0])
```

Comment:
Can you access the file by a name? The index might not be very reliable or desired.

Reply:
not as easily as one would have to filter on `basename` - but this test only needs to verify that self.listing is populated and so `[0]` should be present.